### PR TITLE
Implemented Life Journey Code in Benefit Service

### PR DESCRIPTION
**Pull request by MikeBungay**
Implemented the LifeJourney Endpoint to read from a Json and return information on Life Journey's

### CODE REVIEW

File: handlers/life_journey.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		return c.JSON(http.StatusBadRequest, lifeJourneyResponse)
+	}
+	ljList, err := lifejourneys.LifeJourneyService.LoadLifeJourneys()
+	if err != nil {
+		c.Logger().Error(err)
+		return c.JSON(http.StatusBadRequest, lifeJourneyResponse)
+	}
+
+	if lifeJourneyRequest.Id != 0 {
+		for i, _ := range ljList {
```
Comment (evedube) [inline handlers/life_journey.go]: You should switch this to use the value isntead of i, itll make it easier to read

File: main.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 
 // @title Benefit Service
 // @version 1.0
-// @description This service returns information about benefits
+// @description This service returns information about lifejourneys
```
Comment (evedube) [inline main.go]: This should say benefits as its for the benefit service, I assume it was a find all replace mistake

File: models/LifeJourney.go [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package models
+
+type LifeJourney struct {
+	ID              int      `json:"id"`
+	Title           string   `json:"title"`
+	Description     string   `json:"description"`
+	Title_fr        string   `json:"title_fr"`
```
Comment (evedube) [inline models/LifeJourney.go]: We should try to keep a standard, maybe use TitleFr or TitleFrench to make it easy to read?

File: models/LifeJourney.go [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+	Description_fr  string   `json:"description_fr"`
```
Comment (evedube) [inline models/LifeJourney.go]: Same as above

File: renderings/error.go [hunk 5]
```diff
@@ -0,0 +1,5 @@
+package renderings
+
+type QPSError struct {
```
Comment (evedube) [inline renderings/error.go]: Change this to BenefitServiceError or something of the like

File: server/server.go [hunk 6]
```diff
@@ -11,11 +12,14 @@ import (
 var echoService *echo.Echo
 
 func Main(args []string) {
+	setupLifeJourneyFile()
 	// Echo instance
 	echoService = echo.New()
 	service()
 }
-
+func setupLifeJourneyFile() {
```
Comment (evedube) [inline server/server.go:21]: Not a big deal but adding a space before and after this function will make it easier to read

**Review by evedube — CHANGES_REQUESTED** [review]

File: models/LifeJourney.go [hunk 7]
```diff
@@ -1,10 +1,11 @@
...
-	ID              int      `json:"id"`
-	Title           string   `json:"title"`
-	Description     string   `json:"description"`
-	Title_fr        string   `json:"title_fr"`
-	Description_fr  string   `json:"description_fr"`
-	RelatedBenefits []string `json:"related_benefits"`
+	ID              int            `json:"id"`
+	Title           string         `json:"title"`
+	Description     string         `json:"description"`
+	TitleFr        string         `json:"title_fr"`
```
Comment (evedube) [inline models/LifeJourney.go]: You didnt run go fmt here, add a space and save and your IDE should run it

**Review by evedube — CHANGES_REQUESTED** [review]

**Review by evedube — APPROVED** [review]

**Comment by MikeBungay** [comment]
Merging LifeJourney Code into Dev branch

**Pull request merged**